config/admins.txt

```diff
@@ -136,3 +136,4 @@ Averagejoe82 = Game Master
 The Dreamweaver = Game Master
 88Naoki = Game Master
 Naksuasdf = Game Master
+MrDoomBringer = Game Master

```

Comment:
wtf mrdoombringer trying to take over tg??

Reply:
delete this comment right now PLEASE ILL DO ANYTHING